From b708a56d2efd3c028acb21fb35e2b4b962826874 Mon Sep 17 00:00:00 2001
From: Ranjit Mishra <ranjit.mishra@borqs.com>
Date: Wed, 25 Oct 2017 15:02:39 +0530
Subject: [PATCH] M0065668 : Update metrics daemon socket path

- KaiOS changed the socket path to /dev/socket/metricsd_socket

Change-Id: Ibb8588b9206c49fd4815f2d39268baefbcff9cbd
---

diff --git a/b2g_telephony/NetworkDataCollector.cpp b/b2g_telephony/NetworkDataCollector.cpp
index ab436d9..60ca835 100644
--- a/b2g_telephony/NetworkDataCollector.cpp
+++ b/b2g_telephony/NetworkDataCollector.cpp
@@ -47,9 +47,9 @@
 
 #define NETWORK_PARAMS_LOG_FILE       "/data/data/media/network_parameters.txt"
 
-NS_NAMED_LITERAL_CSTRING(DATA_COLLECTOR_SERVER_SOCKET_NAME, "/system/kaios/metricsd_socket");
+NS_NAMED_LITERAL_CSTRING(DATA_COLLECTOR_SERVER_SOCKET_NAME, "/dev/socket/metricsd_socket");
 
-//#define SERVER_DATA_COLLECTOR_SOCKET_NAME  "/system/kaios/metricsd_socket"
+#define SERVER_DATA_COLLECTOR_SOCKET_NAME  "/dev/socket/metricsd_socket"
 // Amount of time to sleep while waiting for a response to allow yielding of
 // CPU to other threads
 #define SYNC_RSP_SLEEP_USEC 50 * 1000
@@ -603,7 +603,7 @@
            {
                 memset(&serveraddr, 0, sizeof(serveraddr));
                 serveraddr.sun_family = AF_UNIX;
-                strcpy(serveraddr.sun_path,"/system/kaios/metricsd_socket");
+                strcpy(serveraddr.sun_path,SERVER_DATA_COLLECTOR_SOCKET_NAME);
                 QRLOGD(" NetworkDataCollector: socket fd=%d, sockaddress length= %d",mSocket->mFd,sizeof(serveraddr));
                 if( connect(mSocket->mFd, (struct sockaddr *)&serveraddr, sizeof(serveraddr)) < 0 )
                 {
